dashboards/apache-traffic-server/apache-traffic-server.json

```diff
@@ -0,0 +1,1583 @@
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT (latest(global.proxy.process.cache.ram_cache.total_bytes) - latest(global.proxy.process.cache.ram_cache.bytes_used)) / latest(global.proxy.process.cache.ram_cache.total_bytes)*100  AS '(%) free RAM cache' FROM ATSCacheSampleMetrics"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						},
+						"thresholds": [
+							{
+								"alertSeverity": "CRITICAL",
+								"value": 10
+							},
+							{
+								"alertSeverity": "WARNING",
+								"value": 15
+							}
+						]
+					}
+				},
+				{
+					"title": "Total HTTP Incoming Requests",
+					"layout": {
+						"column": 9,
+						"row": 1,
+						"width": 2,
+						"height": 1
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT latest(global.proxy.process.http.incoming_requests) AS 'incoming requests' FROM ATSHttpSampleMetrics"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Number of HTTP Incoming Responses",
+					"layout": {
+						"column": 11,
+						"row": 1,
+						"width": 2,
+						"height": 1
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT latest(global.proxy.process.http.incoming_responses) AS 'incoming responses' FROM ATSHttpSampleMetrics"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "",
+					"layout": {
+						"column": 1,
+						"row": 2,
+						"width": 2,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "\n\n**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/marketplace?account=3510613).\n\nUnable to find data in your dashboard? -[Troubleshoot here](\nhttps://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration/)\n\nFollow New Relic [Apache Traffic Server documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration/) to instrument Apache Traffic Server.\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=ApacheTrafficServer) here and let us know how we can improve it for you.\n"
```

Comment:
```suggestion
						"text": "\n\n**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/marketplace?account=3510613).\n\nUnable to find data in your dashboard? -\nInstrument Apache Traffic Server with New Relic using the [documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration/).\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=ApacheTrafficServer) here and let us know how we can improve it for you.\n"
```

Reply:
Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...